Adding a New Example Notebook

SophiaRowland opened the pull request:
I am adding a new example notebook that walks through creating models in SWAT and XGBOOST and then using SASCTL and PZMM to create the appropriate metadata before pushing everything into Model Manager. This notebook was designed for Viya 2020 environments. I haven't made any changes to the source code - just adding a new example.

jlwalke2 reviewed (changes requested):
Looking good.  Can you clean up the `mm_swat()` method slightly?  The `with` syntax isn't actually needed, and the additional files can simply be passed using the `files=` parameter of `register_model()`.  See example below.

```
model_content = [output_path / f for f in ('dmcas_fitstat.json', 'dmcas_lift.json', 'dmcas_roc.json')]
model_content = {f.name: f for f in model_content}
    
# Pushing Astore into Model Manager
register_model(feature_astore, feature_name, project=project_name, force=True, version='latest')
register_model(model_astore, model_name, project=project_name, force=True, version='latest', files=model_content) 
```

SophiaRowland commented:
Thanks for the suggestions Jonathan! I added hmeq.csv into /data, changed /Data to /data in the notebook and implemented your suggested method in mm_swat().
